mxcube3/routes/Utils.py

```diff
@@ -95,12 +97,142 @@ def my_execute_entry(self, entry):
         foo = ['collectOscillationFinished', 'collectOscillationFailed', 'warning']
         mxcube.collect.emit(random.choice(foo))
     elif isinstance(entry, qe.CharacterisationGroupQueueEntry):
+        logging.getLogger('HWR').info('[QUEUE] Executing CharacterisationGroupQueueEntry entry: %s' %entry)
```

Comment:
I think its better to remove code that is not used, I understand that It might be good to have at some point but you can always get it back with git.